lib/knex-builder/FunctionHelper.js

```diff
@@ -14,25 +14,30 @@ class FunctionHelper {
     return this.client.raw('CURRENT_TIMESTAMP');
   }
 
-  uuidToBin(uuid) {
+  uuidToBin(uuid, flag = true) {
```

Comment:
Can you add some comment to say what is the goal of this flag ? I dont know why is necessary.

Reply:
it just give an option of UUID_TO_BIN(UUID(), false) for mysql  to ensure that the logic of the inserted data is consistent with old data

Reply:
For other reviewers, the doc in mysql about swap flag : https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_uuid-to-bin

Reply:
yep, i think true  as default is better